pkg/scan/filter.go

```diff
@@ -60,17 +64,21 @@ func FilterWithAdvisories(result *Result, advisoryCfgs *configs.Index[advisoryco
 
 	case AdvisoriesSetResolved:
 		resultFindings := lo.Filter(result.Findings, func(finding *Finding, _ int) bool {
-			adv := packageAdvisories[finding.Vulnerability.ID]
-
-			if advisory.IsResolved(adv, result.TargetAPK.Version) {
-				return false
+			adv, ok := packageAdvisories.GetByVulnerabilityID(finding.Vulnerability.ID)
```

Comment:
My 2c: it'd be cool to have a method `GetByAlias` or something like that. It could return a match if the vulnerability ID or an alias matches.

Reply:
IIUC, that's the intent of this method (renamed to drop the `ID` suffix):

```go
// GetByVulnerability returns the advisory that references the given
// vulnerability ID as its advisory ID or as one of the advisory's aliases. If
// such an advisory does not exist, the second return value will be false;
// otherwise it will be true.
```

Reply:
Oh neat - you were one step ahead of me 😎